sdk/java/src/main/java/com/chain/http/Client.java

```diff
@@ -408,15 +408,17 @@ private OkHttpClient buildHttpClient(Builder builder) {
   private static final Random randomGenerator = new Random();
   private static final int MAX_RETRIES = 10;
   private static final int RETRY_BASE_DELAY_MILLIS = 40;
-  private static final int RETRY_MAX_DELAY_MILLIS = 4000;
+
+  // the max amount of time cored leader election could take
+  private static final int RETRY_MAX_DELAY_MILLIS = 15000;
```

Comment:
could this be too much? I think this could cause it to wait 15 seconds between individual retries.